apps/web/src/pages/Departments/ViewDepartmentPage.tsx

```diff
@@ -10,6 +10,7 @@ import {
   Link,
   CardBasic,
   CardSeparator,
+  NoList,
```

Comment:
Lol, this is the first time seeing this component.  :laughing:

Reply:
I asked @yonikid15 sorta jokingly to make it since we often remove styles from the base `ul`. He made it and it is actually useful :stuck_out_tongue: